controllers/dspipeline_params.go

```diff
@@ -548,16 +587,103 @@ func (p *DSPAParams) ExtractParams(ctx context.Context, dsp *dspa.DataSciencePip
 			}
 		}
 
-		// If a Custom CA Bundle is specified for injection into DSP API Server Pod
-		// then retrieve the bundle to utilize during storage health check
+		// Check for Global certs
+		// If it exists, include this cert for tls verifications
+		globalCABundleCFGMapName := config.GlobalODHCaBundleConfigMapName
+		err, globalCerts := util.GetConfigMapValues(ctx, globalCABundleCFGMapName, p.Namespace, client)
+		if err != nil {
+			// If the global cert configmap is not available, that is OK
+			if !apierrs.IsNotFound(err) {
+				log.Info(fmt.Sprintf("Encountered error when attempting to fetch ConfigMap: [%s], Error: %v", globalCABundleCFGMapName, err))
+				return err
+			}
+		} else {
+			// Found a global cert, consume this cert, takes precedence over "cABundle" provided via DSPA
```

Comment:
```suggestion
			// Found a cert provided by odh-operator. Consume it.
```

there is no precedence. All are thrown in a bucket together, and the order doesn't matter.

Reply:
updated!